Orange/widgets/widget.py

```diff
@@ -316,6 +320,12 @@ def save_graph(self):
             return
         saveplot.save_plot(graph_obj, self.graph_writers)
 
+    def __copy_to_clipboard(self):
+        graph_obj = getdeepattr(self, self.graph_name, None)
+        if graph_obj is None:
+            return
+        ClipboardFormat.write_image(None, graph_obj)
```

Comment:
Interesting. Doesn't segfault anymore, yet nothing has changed code-wise. But anyway, now the images are borked:
![distributions](https://cloud.githubusercontent.com/assets/684364/16399924/050324c8-3cd7-11e6-9ae7-27d14bece3ac.png)